common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -87,6 +91,79 @@ def test_uploading_non_csv_file_results_in_error(self):
         self.assertEqual(self.auto_enroll_section.first_notification_message(section_type=self.auto_enroll_section.NOTIFICATION_ERROR), "Make sure that the file you upload is in CSV format with no extraneous characters or rows.")
 
 
+class CSVReportGeneration(UniqueCourseTest):
```

Comment:
My goal here was to make this class sufficiently generic that porting over the tests in `lms/djangoapps/instructor/features/common.py` (which I wrote) would be straightforward. I believe I have done so, but that's out of scope for this PR.